lib/ferto/client.rb

```diff
@@ -164,6 +169,9 @@ def build_body(aggr_id, aggr_limit, url, callback_url, callback_type,
         body[:callback_url] = callback_url
       end
 
+      body[:callback_error_type] = callback_error_type if callback_error_type.present?
+      body[:callback_error_dst] = callback_error_dst if callback_error_dst.present?
```

Comment:
`present?` is a Rails method. There will be a `NoSuchMethod` in these two lines since rails is not used in ferto.

Reply:
you are right, I've tested through Mo, that has rails in place. I'll change that

Reply:
Intresting ruby magic:

```
#!/usr/bin/env ruby

def meth (argumentA: "")
  puts argumentA.class
end

meth
meth(argumentA: "x")
meth(argumentA: nil)

-----------------
String
String
NilClass
```
If `nil` is passed specifically, it does not use the default value!!! 😑